problems/arraysWithLoops.js

```diff
@@ -25,23 +46,45 @@ function shoutWhileLoop() {}
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForOfLoop() {}
+// function shoutForOfLoop() {}
+const shoutForOfLoop = array => {
+  let output = []
+  for(let element of array) {
+    output.push(element + "!")
+  }
+  return output
+}
 
 /**
  * Takes in an array and returns the sum of all values
  * @param {number[]} nums
  * @returns {number} sum
  */
 
-function sumArray() {}
+// function sumArray() {}
+const sumArray = nums => {
+  let sum = 0
+  for(let i = 0; i < nums.length; i++) {
+    sum += nums[i]
+  }
+  return sum
+}
 
 /**
  * Takes in an array and returns an array of all the odd valued elements
  * @param {number[]} nums
  * @returns {array} odds
  */
 
-function oddArray() {}
+// function oddArray() {}
+const oddArray = nums => {
+  let oddnums = []
+    for(let i = 0; i < nums.length; i++) {
+      if(oddnums % 2 === 1)
+      odd.nums.push(nums[1])
```

Comment:
should be oddNums instead of odd.nums 
We don't want to do nums[1] because that never changes we want to do nums[i]